src/other/PropertyEditor.js

```diff
@@ -92,7 +92,7 @@
     PropertyEditor.prototype.enter = function (domNode, element) {
         Accordion.prototype.enter.apply(this, arguments);
         
-        if(!this._parentElement.node().matches(".other_PropertyEditor div")){
+        if(this._parentElement.filter(".other_PropertyEditor div")[0].length === 0){
```

Comment:
Yes, it would. It's a conditional to test whether or not "this" is the outer-most instance of PropertyEditor.